src/MicrosoftTeams.schema.json

```diff
@@ -173,36 +172,107 @@
                         "description": "A unique identifier for the bot that matches its ID in the bot framework.",
                         "maxLength": 64
                     },
+                    "needsChannelSelctor": {
+                        "type": "boolean",
+                        "description": "This value describes whether or not the bot utilizes a user hint to add the bot to a specific channel.",
+                        "default": "false"
+                    },
                     "scope": {
                         "type": "array",
+                        "description": "Specifies whether the bot offers an experience in the context of a channel in a team, or an experience scoped to an individual user alone. These options are non-exclusive",
                         "items": {
                             "enum": [
                                 "team",
-                                "people",
                                 "personal"
-                            ],
-                        "description": "Specifies whether the bot offers an experience in the context of a channel in a team, a chat between one or more people, or an experience scoped to an individual user alone. These options are non-exclusive"
+                            ]
+                        }
+                    },
+                    "commandList": {
+                        "type": "array",
+                        "description": "The list of commands that the bot supplies, including their usage, description, and the scope for which the commands are valid. A seperate command list entry in the array should be used when valid scopes differ",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "scope": {
+                                    "type": "array",
+                                    "description": "Specifies the scopes for which the command list is valid",
+                                    "items": {
+                                        "enum": [
+                                            "team",
+                                            "personal"
+                                        ]
+                                    }
+                                },
+                                "commands": {
+                                    "type": "array",
```

Comment:
I'd defer to @larryjin or @richardtaylorrt on that max as it depends on the ui here.